js/bybit.js

```diff
@@ -4265,93 +4890,224 @@ module.exports = class bybit extends Exchange {
         //         "time": "1666768215157"
         //     }
         //
+        const result = this.safeValue (response, 'result', {});
+        const trades = this.safeValue (result, 'list', []);
+        return this.parseTrades (trades, market, since, limit);
+    }
+
+    async fetchMyDerivativesTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchMyDerivativesTrades() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            // 'orderId': 'f185806b-b801-40ff-adec-52289370ed62', // if not provided will return user's trading records
+            // 'startTime': parseInt (since / 1000),
+            // 'endTime': 0,
+            // 'category': ''
+            // 'limit' 20, // max 50
+        };
+        if (since !== undefined) {
+            request['startTime'] = since;
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit; // default 20, max 50
+        }
+        const response = await this.privateGetUnifiedV3PrivateExecutionList (this.extend (request, params));
```

Comment:
Hum this is not working:
![image](https://user-images.githubusercontent.com/43336371/204506074-9c637f00-4db4-4170-9c6f-f98790bf6123.png)

I think we're missing the `category` 
![image](https://user-images.githubusercontent.com/43336371/204506234-84659e9f-5598-4ccf-820c-978834146ebf.png)